src/components/media-types/video/index.js

```diff
@@ -49,37 +47,33 @@ export const VideoComponent = ({
     }
   }, [inView])
 
-  if (displayView) {
-    return (
-      <div className={styles.video}>
-        <video
-          ref={domElement}
-          className={styles.displayviewVideo}
-          autoPlay={inView}
-          playsInline
-          muted
-          loop
-          controls={interactive}
-          src={preview ? previewUri : artifactUri}
-          poster={displayUri}
-        />
-      </div>
-    )
-  } else {
-    return (
-      <>
-        <video
-          ref={domElement}
-          className={styles.video}
-          autoPlay={inView}
-          playsInline
-          muted
-          loop
-          controls={interactive}
-          src={preview ? previewUri : artifactUri}
-          poster={displayUri}
-        />
-      </>
-    )
-  }
+  return displayView ? (
+    <div className={styles.video}>
+      <video
+        ref={domElement}
+        className={styles.displayviewVideo}
+        autoPlay={inView}
+        playsInline
+        muted
+        loop
+        controls={interactive}
+        src={preview ? previewUri : artifactUri}
+        poster={displayUri}
+      />
+    </div>
+  ) : (
+    <>
```

Comment:
not important, but the `<></>` wrapping is not needed (i know, it was there before)